perfkitbenchmarker/linux_benchmarks/cloud_bigtable_ycsb_benchmark.py

```diff
@@ -229,6 +232,21 @@ def Prepare(benchmark_spec):
   benchmark_spec.always_call_cleanup = True
   vms = benchmark_spec.vms
 
+  if FLAGS.google_bigtable_cluster_name is None:
+    cluster_name = 'pkb-bigtable-{0}'.format(FLAGS.run_uri)
+    project = FLAGS.project or _GetDefaultProject()
+    logging.info('Creating bigtable cluster %s', cluster_name)
+    subprocess.check_call([FLAGS.gcloud_path, 'alpha', 'bigtable', 'clusters',
```

Comment:
It might be worth extracting this into a class inheriting from `resource.BaseResource`, which will handle retries if the calls fail, and gracefully handle deletes.